CHANGELOG.md

```diff
@@ -6,11 +6,11 @@ Notable changes to this project are documented in this file. The format is based
 
 Breaking changes:
 
-New features:
```

Comment:
We haven't updated the CHANGELOG for these `purescm` versions. When there is a new version of the upstream package we get those changelog items.